store/tikv/txn_committer_test.go

```diff
@@ -118,3 +118,48 @@ func (s *testCommitterSuite) TestCommitRollback(c *C) {
 		"c": "c2",
 	})
 }
+
+func (s *testCommitterSuite) TestPrewriteRollback(c *C) {
+	s.mustCommit(c, map[string]string{
+		"a": "a0",
+		"b": "b0",
+	})
+
+	txn1 := s.begin(c)
+	err := txn1.Set([]byte("a"), []byte("a1"))
+	c.Assert(err, IsNil)
+	err = txn1.Set([]byte("b"), []byte("b1"))
+	c.Assert(err, IsNil)
+	committer, err := newTxnCommitter(txn1)
+	c.Assert(err, IsNil)
+	err = committer.prewriteKeys(NewBackoffer(prewriteMaxBackoff), committer.keys)
+	c.Assert(err, IsNil)
+
+	txn2 := s.begin(c)
+	v, err := txn2.Get([]byte("a"))
+	c.Assert(err, IsNil)
+	c.Assert(v, BytesEquals, []byte("a0"))
+
+	err = committer.prewriteKeys(NewBackoffer(prewriteMaxBackoff), committer.keys)
+	if err != nil {
```

Comment:
It was nil before the bug been fixed. I do not use `assert` here because I want to prove the bug will cause staled read at L164.